### ISSUE TITLE
Option to not format double curly braces, including inside tags

### ISSUE THREAD

**Issue by xerroxcopy** [prettier/plugin-pug#103]
## Request / Idea

The idea is almost identical to #98, but with extension to tags. I'm very sorry that I have missed your point at 'more than just text-tokens'. You were dead right. I actually needed double curly braces inside tags too. Example from [Stein Expedite docs](https://docs.steinhq.com/expedite-display-data):

```html
<div
  data-stein-url="https://api.steinhq.com/v1/storages/5cc158079ec99a2f484dcb40/Sheet1"
  data-stein-limit="2"
>
  <div>
    <h1>{{title}}</h1>
    <h6>By {{author}}</h6>
    <p>
      {{content}}
    </p>
    <p>Read on <a href="{{link}}">Medium</a></p>
  </div>
</div>
```
the `{{link}}` inside `<a>` should also be formatted as is, not as `{{ link }}`.

## Input

```pug
a(href="{{link}}") Link
```

## Expected Output

```pug
a(href="{{link}}") Link
```

## Current behaviour

```pug
a(href="{{ link }}") Link
```

**Comment by Shinigami92** [comment]
There they are, the test cases 😜

Sry for beeing a bit like MS Windows: Only add features and fix bugs that are found by users 😛

Hope I have time for this next hours/days

**Comment by Shinigami92** [comment]
@xerroxcopy Are there any other parts that needs to be covered?
Pls have a look at https://github.com/prettier/plugin-pug/pull/104/files#diff-4f50bb302d1c998ef11b403520e97609

**Comment by xerroxcopy** [comment]
I... guess that is enough, but to be bloody honest, I have no idea. Just as MS Windows users, what I can point out is what I only need right now :p I am very sorry. I won't use them as tag names (e.g., `{{tag}}.class-one`), I won't use them inside scripts, while others might. Still, I believe attributes is enough.

**Comment by Shinigami92** [comment]
Released as [1.5.1](https://www.npmjs.com/package/@prettier/plugin-pug/v/1.5.1)  
Have a nice weekend 👋
